front/app/modules/commercial/insights/services/insightsCategoriesSuggestionsTasks.ts

```diff
@@ -47,6 +50,8 @@ export async function insightsTriggerCategoriesSuggestionsTasks(
   categories?: string[],
   inputs?: string[]
 ) {
+  const pollingStream = of({}).pipe(delay(3000), repeat());
```

Comment:
Not an expert at rxjs, but suspected there would be a native concept for this and there [seems to be one indeed](https://www.learnrxjs.io/learn-rxjs/operators/creation/interval).

Reply:
Excellent suggestion, looks much cleaner now! thanks 🙌